test/sequence.js

```diff
@@ -8,29 +8,29 @@ var eq = require('./shared/eq');
 describe('sequence', function() {
 
   it('operates on a list of lists', function() {
-    eq(R.sequence(R.of, []), [[]]);
-    eq(R.sequence(R.of, [[], [1, 2, 3, 4]]), []);
-    eq(R.sequence(R.of, [[1], [2, 3, 4]]), [[1, 2], [1, 3], [1, 4]]);
-    eq(R.sequence(R.of, [[1, 2], [3, 4]]), [[1, 3], [1, 4], [2, 3], [2, 4]]);
-    eq(R.sequence(R.of, [[1, 2, 3], [4]]), [[1, 4], [2, 4], [3, 4]]);
-    eq(R.sequence(R.of, [[1, 2, 3, 4], []]), []);
+    eq(R.sequence(Array, []), [[]]);
+    eq(R.sequence(Array, [[], [1, 2, 3, 4]]), []);
+    eq(R.sequence(Array, [[1], [2, 3, 4]]), [[1, 2], [1, 3], [1, 4]]);
+    eq(R.sequence(Array, [[1, 2], [3, 4]]), [[1, 3], [1, 4], [2, 3], [2, 4]]);
+    eq(R.sequence(Array, [[1, 2, 3], [4]]), [[1, 4], [2, 4], [3, 4]]);
+    eq(R.sequence(Array, [[1, 2, 3, 4], []]), []);
```

Comment:
Damn, this was *always* better!

Reply:
I agree. Type representatives provide a tidy home for ancillary functions such as `Array#fantasy-land/of`.